Update accessibility statement

tombye opened the pull request:
Pull request to update the review dates and those for fixing the issues listed.

> In draft because we intend to discuss and change the dates for fixing the issues on Github

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -13,8 +13,8 @@ <h1 class="heading-large">Accessibility statement</h1>
-      "Last updated": "27 April 2022",
-      "Next review due": "12 July 2022"
+      "Last updated": "4 August 2022",
+      "Next review due": "3 November 2022"
```
Should we try to keep the reviews in line with quarters, so we’re doing it around the same time as we update the roadmap?

tombye replied:
Good idea. I was slightly guessing on the exact dates here. What dates would you suggest?

karlchillmaid replied:
```suggestion
      "Next review due": "20 September 2022"
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
     </li>
     <li>
-      expanding and collapsing groups of checkboxes using a screen reader can be confusing
+      Expanding and collapsing groups of checkboxes using a screen reader can be confusing. We expect/hope/plan to fix this by 7 April 2022.
```
```suggestion
      Expanding and collapsing groups of checkboxes using a screen reader can be confusing. We expect/hope/plan to fix this by 7 April 2023.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
     </li>
     <li>
-      choosing the date and time you want to send a message using a screen reader can be confusing
+      Choosing the date and time you want to send a message using a screen reader can be confusing. We expect/hope/plan to fix this by 17 March 2022.
```
```suggestion
      Choosing the date and time you want to send a message using a screen reader can be confusing. We expect/hope/plan to fix this by 17 March 2023.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
     </li>
     <li>
-      our validation errors cause problems for screen readers - this means it can be hard to know when an error has occurred or what to do next
+      Our validation errors cause problems for screen readers. This means it can be hard to know when an error has occurred or what to do next. We expect/hope/plan to fix this by 05 12 2022.
```
```suggestion
      Our validation errors cause problems for screen readers. This means it can be hard to know when an error has occurred or what to do next. We plan to fix this by 5 December 2022.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
+  <ol class="govuk-list govuk-list--number">
     <li>
-      the JAWS screen reader announces template folder page headings as multiple sentences
+      The JAWS screen reader announces template folder page headings as multiple sentences. We expect/hope/plan to fix this by 13 January 2022.
```
```suggestion
      The JAWS screen reader announces template folder page headings as multiple sentences. We plan to fix this by 13 January 2023.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
     </li>
     <li>
-      some icons do not appear when using high contrast mode on Windows - this fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>
+      Some icons do not appear when using high contrast mode on Windows. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>. We expect/hope/plan to fix this by 17 February 2022.
```
```suggestion
      Some icons do not appear when using high contrast mode on Windows. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>. We plan to fix this by 17 February 2023.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
     </li>
     <li>
-      the JAWS and NVDA screen readers do not announce the ‘copy to clipboard’ button - this fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">success criterion 4.1.3: status messages</a>
+      The JAWS and NVDA screen readers do not announce the ‘copy to clipboard’ button. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">success criterion 4.1.3: status messages</a>. We expect/hope/plan to fix this by 27 January 2022.
```
```suggestion
      The JAWS and NVDA screen readers do not announce the ‘copy to clipboard’ button. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">success criterion 4.1.3: status messages</a>. We plan to fix this by 27 January 2023.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
+  <ol class="govuk-list govuk-list--number">
     <li>
-      the <abbr title="HyperText Markup Language">HTML</abbr> code for the ‘upload a file’ button may cause issues for assistive technology users - this fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships">success criterion 1.3.1: info and relationships</a>
+      The <abbr title="HyperText Markup Language">HTML</abbr> code for the ‘upload a file’ button may cause issues for assistive technology users. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships">success criterion 1.3.1: info and relationships</a>. We expect/hope/plan to fix this by 12 December 2022.
```
```suggestion
      The <abbr title="HyperText Markup Language">HTML</abbr> code for the ‘upload a file’ button may cause issues for assistive technology users. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships">success criterion 1.3.1: info and relationships</a>. We expect to fix this by 12 December 2022.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -105,55 +105,39 @@ <h2 class="heading-medium" id="non-accessible-content">Non-accessible content</h
     </li>
     <li>
-      the way we label the different regions of our pages is confusing for screen reader users
+      The way we label the different regions of our pages is confusing for screen reader users. We expect/hope/plan to fix this by 30 September 2022.
```
```suggestion
      The way we label the different regions of our pages is confusing for screen reader users. We expect to fix this by 30 September 2022.
```

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -13,8 +13,8 @@ <h1 class="heading-large">Accessibility statement</h1>
       "Published": "23 September 2020",
-      "Last updated": "27 April 2022",
-      "Next review due": "12 July 2022"
+      "Last updated": "4 August 2022",
```
```suggestion
      "Last updated": "9 August 2022",
```

tombye replied:
Realistically these changes will go out tomorrow rather than today so I'm going to ignore the suggestion in favour of using tomorrow's date.

karlchillmaid reviewed (approved):
Looks good, Tom. Thanks for reordering all the things.

karlchillmaid commented on app/templates/views/accessibility_statement.html:
```diff
@@ -13,8 +13,8 @@ <h1 class="heading-large">Accessibility statement</h1>
       "Published": "23 September 2020",
-      "Last updated": "27 April 2022",
-      "Next review due": "12 July 2022"
+      "Last updated": "4 August 2022",
```
```suggestion
      "Last updated": "16 August 2022",
```

tombye replied:
I was away yesterday so couldn't deploy the changes, making this date wrong. I've updated it to today in https://github.com/alphagov/notifications-admin/pull/4309/commits/26c50f9bba0b1c4eb7728358b9770d16ae5b2f88 so marking this as resolved.

tombye commented:
@karlchillmaid I've changed the 'last updated' date to today and removed the issue with file uploads because it's been fixed since we started this PR 🚀. The whole thing looks like this now:

![image](https://user-images.githubusercontent.com/87140/185105209-14b5ae82-9988-4853-b00f-c92c136821ea.png)

tombye commented:
Chatting off-Github, @karlchillmaid is happy with the changes made since his initial approval.
